source/_integrations/hue_ble.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: Hue BLE
+description: Instructions on how to setup Phillips Hue Bluetooth lights within Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.7
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@flip-dots'
+ha_domain: hue_ble
+ha_platforms:
+  - light
+ha_bluetooth: true
+ha_integration_type: integration
+---
+
+The **HueBLE** {% term integration %} allows you to control your Phillips Hue Bluetooth lights with Home Assistant.
```

Comment:
```suggestion
The **Hue BLE** {% term integration %} allows you to control your Phillips Hue Bluetooth lights with Home Assistant.
```
Should this be Hue BLE? At least that is what it is called in the title.

Reply:
I think I meant "Hue BLE" but to be honest im not sure myself, I prefer "HueBLE" for when im refering to it in code but I think "Hue BLE" looks a little better when presented in a UI.